pkg/cli/delete/types.go

```diff
@@ -0,0 +1,39 @@
+/*
+Copyright 2025 The Radius Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package delete
+
+import (
+	"context"
+
+	"github.com/radius-project/radius/pkg/cli/clients"
+)
+
+//go:generate mockgen -typed -destination=./mock_delete.go -package=delete -self_package github.com/radius-project/radius/pkg/cli/delete github.com/radius-project/radius/pkg/cli/delete Interface
+
+// Interface is the interface for executing Bicep deployments in the CLI.
+type Interface interface {
```

Comment:
The interface comment mentions "Bicep deployments" but this is for delete operations. Consider updating the comment to accurately reflect the delete functionality.